src/components/existing-request/existing-request-search.vue

```diff
@@ -119,9 +120,11 @@ export default class ExistingRequestSearch extends Vue {
 
   mounted () {
     this.$nextTick(() => {
-      // add classname to button text (for more detail in Sentry breadcrumbs)
-      const retrieveNrBtn = this.$el.querySelector("#retrieve-name-btn > span")
-      if (retrieveNrBtn) retrieveNrBtn.classList.add("retrieve-nr-btn")
+      if (this.$el?.querySelector) {
```

Comment:
This page, and a couple others, were missing this check... which I still think should not be necessary, but Sentry has reported errors here... 🤷‍♂️

Reply:
BTW, it's not a big deal if this classname code doesn't run, as it's only to decorate the buttons for easier trouble-shooting in Sentry.

However, it is a big deal if code causes an error/exception, because nothing else in this method will run.